docker-build/baseball.sql

```diff
@@ -0,0 +1,42 @@
+use baseball;
+#select * from game limit 10;
+
+drop table if exists batting_avg_wDATEDIFF ;
```

Comment:
Don't drop the tables! instead you should use "create table if not exists" to make the intermediate tables. That way if the tables still exist from the last calculation it will run faster.